flink-core/src/test/java/org/apache/flink/api/common/typeutils/base/array/DoublePrimitiveArraySerializerTest.java

```diff
@@ -25,7 +25,7 @@
  * A test for the {@link
  * org.apache.flink.api.common.typeutils.base.array.LongPrimitiveArraySerializer}.
  */
```

Comment:
Probably not related to the main PR's goal however shouldn't it be
```suggestion
 * A test for the {@link DoublePrimitiveArraySerializer}.
 */
```
?